### ISSUE TITLE
Sergei Strelec Integration Problem

### ISSUE THREAD

**Issue by AnomSanjaya** [nguyentumine/AIO-Boot#60]
Hi, i got problem about integration of Sergei Strelec on newest AIO-Boot version.

TLDR:
1. Update a AIO-Boot from v0.9.8.17 to v0.9.9.2.
2. I do reinstalling GRUB bootloader (on GPT UEFI & GPT Leagcy mode).
3. All WinPE pack where i use in below makes bricked on Legacy & UEFI mode.
4. I do reintegrating Macrium Reflect WinPE at first.
5. I do test booting and success (Legacy & UEFI Mode).  
6. I do reintegrating Sergei Strelec WinPE at second.
7. I found the strange here, thats no more Sergei Strelec WinPE at MENU.
![image](https://user-images.githubusercontent.com/31486344/90026115-7d8b6700-dce9-11ea-9681-0e684abbbaa3.png)
8. I decide to add manually Sergei Strelec WinPE on Legacy and UEFI mode.
![image](https://user-images.githubusercontent.com/31486344/90026357-c7744d00-dce9-11ea-852c-a216dcba593c.png)
![image](https://user-images.githubusercontent.com/31486344/90026459-e541b200-dce9-11ea-9cbe-d34c734fe4f9.png)
9. Sergei Strelec WinPE still none to show in MENU (pict same as number 7).
10. I do test booting in UEFI Mode and success .
![IMG_2020-08-12-22-24-14-750](https://user-images.githubusercontent.com/31486344/90027132-c55ebe00-dcea-11ea-92d8-c9658827829e.jpg)
11. I do test booting in Legacy mode and only showing Macrium Reflect WinPE here.
![IMG_2020-08-12-22-22-54-469](https://user-images.githubusercontent.com/31486344/90027284-f3dc9900-dcea-11ea-98ab-ab2f51052774.jpg)
12. The Sergei Strelec menu in Legacy mode isn't working properly.

*I have 500 GB HDD external (GPT Style Partition)
*I use 2 WinPE (Macrium Reflect & Sergei Strelec)
*I'm testing in real machine (Acer One 14 Z1401 & Lenovo ThinkPad X240) and same result
*Bonus: Indonesian Language will uploading soon

**Comment by nguyentumine** [comment]
Strelec entry in the main menu.  it is no longer located in the WinPe & Setup menu.
Booting Windows Boot Manager in Legacy mode on GPT disks is not supported.
Try using [grubfm](https://github.com/a1ive/grub2-filemanager) to boot Strelec's WIM files.  If the issues are the same, report it to him. Read /AIO/grubfm/Readme.txt

**Comment by AnomSanjaya** [comment]
The Sergei Strelec was succeeded booting in GrubFM mode, except Win8NE (Native Edition).

In Win8NE, i got error code 0xc0000428 as BOOTMGR bootloader problem.

**Comment by nguyentumine** [comment]
@AnomSanjaya In the aioboot menu, what's not working?  the Strelec menu is on the main menu, not in WinPE & Setup

**Comment by AnomSanjaya** [comment]
The Sergei Strelec menu in AIO-Boot Menu isn't working bcoz the required file (mainly GRLDR) is missing or not included when integrating process.

**Comment by nguyentumine** [comment]
@AnomSanjaya DOS files will not be integrated.  you can manually integrate with AIOCreator

**Comment by AnomSanjaya** [comment]
> 
> 
> DOS files will not be integrated. you can manually integrate with AIOCreator

This makes me integrate it one-by-one (per DOS apps)


How about copying some require files for booting DOS to SSTR folder and will be result like this
![gambar](https://user-images.githubusercontent.com/31486344/90259962-6da48c00-de7d-11ea-8b97-47f4c4d421e9.png)

![gambar](https://user-images.githubusercontent.com/31486344/90260715-7ba6dc80-de7e-11ea-9402-c55d20b32c0d.png)

*picture when working in real machine will uploading ASAP

**Comment by nguyentumine** [comment]
New version is available, just re-integrate the SSTR into the AIO Boot drive.

**Comment by AnomSanjaya** [comment]
Im already update to latest version and will be result:

UEFI mode: same problem, nothing to show the Strelec Win10 x64, except iim must adding it manually in "Win Menu?"
Legacy mode : grldr successful loaded but, only "Boot USB Sergei Strelec" isn't working and Strelec WinPE didn't show in WinPE & Setup

**Comment by AnomSanjaya** [comment]
Update: cause problem is out-to-date of script on sergei strelec
![image](https://user-images.githubusercontent.com/31486344/92296960-de1a6680-ef6c-11ea-8481-8329a55dbdb0.png)

![image](https://user-images.githubusercontent.com/31486344/92296928-94318080-ef6c-11ea-9fbe-b79e29964203.png)


for everyone who faced these problem, delete the sergei srelec old script in \AIO\Menu\Main.cfg and re-integrate again
